src/Common/Session/SessionUtil.php

```diff
@@ -50,26 +50,27 @@ class SessionUtil
     private static $use_cookies = true;
     private static $use_only_cookies = true;
 
-    public static function coreSessionStart($web_root)
+    public static function coreSessionStart($web_root, $sessionTask = null): void
     {
         if (version_compare(phpversion(), '7.3.0', '>=')) {
             session_start([
+                'read_and_close' => $sessionTask === 2,
```

Comment:
just noted this wasn't in the below session_start when using php < 7.3.0

Reply:
been so focused on PHP7.3 lately, i've kinda ignored it's step children.:) thanks.
Today i'm back to CDR in hopes of eking out some more enhancement there.
Maybe a few more ajax/fetch replacements too as some examples for modernization team.
@tywrenn Is this part of your teams scope(ajax replace)?

Reply:
@sjpadgett I'm a CSS and SCSS person, not an ajax person 😂 so I dunno what ur talking about

Reply:
lol, my Dad told me once to never tell anybody you know how to work on cars, own a truck and in my case, know about computers! Be surprised how many friends you have once that's out.:)
Ajax! what's ajax????:)

Reply:
@sjpadgett , I am really liking this mechanism since it seems much more natural and explicit than mechanism 1 (my vote would be to only use this mechanism to keep things simple). What would be your thoughts of always setting this and then adding a flag in front of globals.php to "allow" the script to modify session variables. Guessing the number of scripts that need to write to session are small and that most that are doing it, shouldn't be (ie. a crutch that needs to be removed). This would add another layer of security and really package things up nicely as try to work OpenEMR out of the "need to modify the session cookie" hole.

For example, even in globals, on very few scripts need to set session for site_id, language_direction, pid, loginfailure(auth.inc), last_update(auth.inc; this is something we would need to solve btw since scripts that do not allow session writes would not be able to use this mechanism) 

This would be the technique of rip the bandage off :)

Reply:
@bradymiller, Agreed and makes the best sense. Not as safe as my approach of easing in but, I like the cut of your jib.:) With some grep magic, this could go smoother than we could have hoped.
I'll go ahead and revert my flags to get us ready.